src/components/FarmDualCard/FarmDualCardDetails.tsx

```diff
@@ -323,157 +335,212 @@ const FarmDualCardDetails: React.FC<{
       ? '< $0.001'
       : '$' + earnedUSD.toLocaleString();
 
+  const valueOfTotalStakedAmountInUSDC =
```

Comment:
booleans should be named in a readable manner e.g `hasValueOfTotalStakedAmountInUSDC`

Reply:
`valueOfTotalStakedAmountInUSDC` is not boolean. This is USD value of total stake amount of the staking info with `TokenAmount` type